com.ibm.wala.util/src/main/java/com/ibm/wala/util/graph/impl/BasicGraph.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) 2002 - 2021 IBM Corporation.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
```

Comment:
Why is this an appropriate attribution for brand-new code written by a non-IBM employee?